photon-client/src/components/cameras/CameraCalibrationCard.vue

```diff
@@ -384,7 +384,7 @@ const endCalibration = () => {
               color="secondary"
               style="width: 100%"
               :disabled="!settingsValid"
-              @click="isCalibrating ? useCameraSettingsStore().takeCalibrationSnapshot(true) : startCalibration()"
```

Comment:
Oh yeah this fun hack. We should probably not have one button doing two things

Reply:
nah its chill, the logic makes sense now with better variable names